CHANGES.md

```diff
@@ -0,0 +1,79 @@
+# Changes
```

Comment:
Can we call it `CHANGELOG` instead of  `CHANGES` and also change the file name to reflect it?